is-edge.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = function isEdge(ua) {
+  ua = ua || navigator.userAgent;
+  return ua.indexOf('Edge/') > -1;
```

Comment:
I updated this to use `!==` just for consistency's sake. As a pointless aside, this did make me wonder whether a `> -1` check is more performant than `!== -1` for CPUs.